tests/utils.js

```diff
@@ -738,6 +739,20 @@ module.exports = {
       .catch(done.fail);
   },
 
+  seedTestDataNative: async (userContactDoc, documents) => {
+    try {
+      documents.forEach(async doc => await module.exports.saveDocNative(doc));
+    } catch (err) {
+      console.log('error saving docs');
+    }
+    
+    const existingContactDoc = await module.exports.getDocNative(constants.USER_CONTACT_ID);
+    if (userContactDoc) {
+      Object.assign(existingContactDoc, userContactDoc);
+      await module.exports.saveDocNative(existingContactDoc);
```

Comment:
`Object.assign` will mutate the existing object `existingContactDoc` which is fine here right now, but generally undesirable. It can lead to bugs over time where if a future developer comes along and adds code down after this conditional block and doesn't realize that `userContactDoc` may change based on some logic above, they can't be sure of the state. The preferred approach is to treat objects as immutable (although they aren't really) and just create a new object. This handy syntax makes that easy:
```suggestion
      const mergedContact = { ...existingContactDoc, ...userContactDoc };
      await module.exports.saveDocNative(mergedContact);
```
....that could all be on a single line too, but I think the named variable here makes the intention clear and easier to scan when reading.

Reply:
@craig-landry  I have updated to use the recommended syntax. Need a re-review.